setup-gcloud-base/src/setup-gcloud.js

```diff
@@ -49,29 +49,33 @@ const configureGcloud = async (serviceAccountKey, exportCredentials) => {
   return projectId;
 };
 
-const setupGcloud = async (serviceAccountKey, version = 'latest', exportCredentials = false) => {
+const setupGcloud = async (serviceAccountKey, cacheGcloud, version = 'latest', exportCredentials = false) => {
   let semver = version;
   if (!semver || semver === 'latest') {
     semver = await getLatestVersion();
     core.info(`Use latest gcloud version: ${semver}`);
   }
   const downloadUrl = getDownloadUrl(semver);
 
-  return loadTool({
-    tool: 'gcloud',
-    binary: 'google-cloud-sdk',
-    version: semver,
-    downloadUrl,
-  }).then((gcloud) => {
+  let gcloud;
+  if (cacheGcloud) {
```

Comment:
Is this correct? If the cache is hit, is that the time to install? If cache is not hit, are we not supposed to install the tool?

Reply:
yeah, I think this needs to be `!()`